packages/ui-app/src/AddressSummary.tsx

```diff
@@ -2,20 +2,21 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { I18nProps } from './types';
-
-import BN from 'bn.js';
-import React from 'react';
 import { AccountId, AccountIndex, Address } from '@polkadot/types';
+import BaseIdentityIcon from '@polkadot/ui-identicon';
```

Comment:
Move alphabetical -

- types go first (anything `@polkadot/<package/types` or `../types`, etc.
- blank line
- external non-polkadot imports
- polkadot imports
- blank line
- LOCAL IMPORTS

Reply:
I hope I got that right, so _external non-polkadot imports_ and _polkadot imports_ are not mixed, but they are **not** separated by a blank like eithr (e.g how it basically way before I messed things up). I'll do that and you can still tell me if that's wrong.